samples/ControlCatalog/Pages/ToolTipPage.xaml

```diff
@@ -1,22 +1,34 @@
 <UserControl xmlns="https://github.com/avaloniaui">
-  <StackPanel Orientation="Vertical" Gap="4">
-    <TextBlock Classes="h1">ToolTip</TextBlock>
-    <TextBlock Classes="h2">A control which pops up a hint when a control is hovered</TextBlock>
+    <StackPanel Orientation="Vertical"
+                Gap="4">
+        <TextBlock Classes="h1">ToolTip</TextBlock>
+        <TextBlock Classes="h2">A control which pops up a hint when a control is hovered</TextBlock>
 
-    <StackPanel Orientation="Horizontal"
-              Margin="0,16,0,0"
-              HorizontalAlignment="Center"
-              Gap="16">
-      <Border Background="{StyleResource ThemeAccentBrush}"
-              Padding="48,48,48,48">
-        <ToolTip.Tip>
-          <StackPanel>
-            <TextBlock Classes="h1">ToolTip</TextBlock>
-            <TextBlock Classes="h2">A control which pops up a hint when a control is hovered</TextBlock>
-          </StackPanel>
-        </ToolTip.Tip>
-        <TextBlock>Hover Here</TextBlock>
-      </Border>
+        <StackPanel Orientation="Horizontal"
+                    Margin="0,16,0,0"
+                    HorizontalAlignment="Center"
+                    Gap="16">
+            <CheckBox IsChecked="{Binding ElementName=Border, Path=(ToolTip.IsOpen)}"
+                      Content="Show ToolTip" />
+            <Border Name="Border"
+                    Background="{StyleResource ThemeAccentBrush}"
+                    Margin="5"
+                    Padding="50"
+                    ToolTip.Placement="Bottom">
+                <ToolTip.Tip>
+                    <StackPanel>
+                        <TextBlock Classes="h1">ToolTip</TextBlock>
+                        <TextBlock Classes="h2">A control which pops up a hint when a control is hovered</TextBlock>
+                    </StackPanel>
+                </ToolTip.Tip>
+                <TextBlock>Hover Here</TextBlock>
+            </Border>
+            <Border Background="{StyleResource ThemeAccentBrush}"
+                    Margin="5"
+                    Padding="50"
+                    ToolTip.Tip="Another tip">
+                <TextBlock>And Here</TextBlock>
+            </Border>
+        </StackPanel>
```

Comment:
As a demo I find this to be quite confusing. Is there a better way of demonstrating this?

Reply:
I'm open to suggestions. I wanted to have two tool tips, one relative to the target, that can be triggered by the check box, and another that's relative to the mouse. Also wanted to check how well moving between two tool tips works.

Reply:
Not sure, but how about this?

- Have the default behavior border/tooltip first
- Put the "Show ToolTip" checkbox _below_ the (now second) border and change its text to "ToolTip Open"

That might be clearer, what do you think?

Reply:
Sure, makes sense